warp10/src/main/java/io/warp10/continuum/gts/GTSHelper.java

```diff
@@ -2591,14 +2607,20 @@ public static GTSEncoder parse(GTSEncoder encoder, String str, Map<String,String
       idx2 = UnsafeString.indexOf(str, '}', idx);
       
       if (-1 == idx2){
-        throw new ParseException(str, idx);
+        throw new ParseException("Missing end of labels '}'.", str.length() - 1);
```

Comment:
Invalid line no longer shown.

Reply:
Now that we have a mechanism to control the error message size I think it would be a good idea to include the invalid line whenever possible, it makes the diagnosis of the issue way easier.